components/MultipleChoiceList.js

```diff
@@ -46,26 +45,29 @@ class MultipleChoiceList extends React.Component {
     var template = this.props.template;
 
     return (
-      <ul className={this.props.cssClass}>
+      <div className={this.props.rootClass}>
         {facetValues.map(facetValue => {
           return (
-            <li key={facetValue.name} onClick={this.handleClick.bind(this, facetValue.name)}>
+            <div className={this.props.itemClass} key={facetValue.name} onClick={this.handleClick.bind(this, facetValue.name)}>
               <Template data={facetValue} template={template} />
-            </li>
+            </div>
           );
         })}
-      </ul>
+      </div>
     );
   }
 }
 
 MultipleChoiceList.propTypes = {
-  cssClass: React.PropTypes.oneOfType([
+  rootClass: React.PropTypes.oneOfType([
+    React.PropTypes.string,
+    React.PropTypes.array
```

Comment:
If we know the type of the elements of the array, we can use arrayOf instead array, like this (if we think that we want an array of string) : 
```js
 React.PropTypes.arrayOf(React.PropTypes.string)
```